src/vellum/workflows/exceptions.py

```diff
@@ -35,7 +35,7 @@ class WorkflowInitializationException(Exception):
     def __init__(
         self,
         message: str,
-        workflow_definition: Type["BaseWorkflow"],
+        workflow_definition: Optional[Type["BaseWorkflow"]],
```

Comment:
default this kwarg to `None`. Below, set `self.workflow_definition = import_workflow_class()` if `workflow_definition` is `None`